projects/go-lib/src/lib/components/go-side-nav/go-side-nav/go-side-nav.component.scss

```diff
@@ -66,8 +67,18 @@
     display: flex;
   }
 
+  &:hover {
+    background: $theme-dark-bg-hover;
+  }
+
+  &:active {
+    background: $theme-dark-bg-active;
+  }
+
+
   .go-side-nav__app-drawer--icon-collapsed {
     display: flex;
+    font-size: 24px;
```

Comment:
In Goponents, we use `rem`s for font-sizes. Can you change this to `2rem` instead?

Reply:
> In Goponents, we use `rem`s for font-sizes. Can you change this to `2rem` instead?

Shouldnt 24px equal 1.5rem?

Reply:
Yesssss, sorry. Math is hard sometimes lol.

Reply:
Yes please adjust the positioning to be aligned correctly. Ideally the app drawer icon should be centered to the side navigation icons so that when the side navigation is collapsed all the icons are centered.

Reply:
@kpmurphh : When the side nav is collapsed, do we want the app drawer icon to shrink?
The implementation right now shrinks the icon when side nav is collapsed.
Although I dont see any such rule in the DSM. Even the side nav icons stay the same in size when side nav is collapsed.
Can you please confirm?

Reply:
Also, when the side nav is collapsed, the hamburger icon has hover state, the background of navigation icons (i.e. the entire bar containing a navigation icon) has a hover state.
Going by this design language, shouldnt the app drawer icon also have a hover state (only when the side nav is collapsed)?
Because, there is an action supported by the app drawer icon when side nav is collapsed. The app drawer icon open the action sheet, which otherwise the 'more' icon would do in case side nav is expanded.
Your views: @grahamhency , @kpmurphh ?

Reply:
Thanks @kpmurphh .
@grahamhency : Just pushed the changes, PR is ready for review. ✌️